frontend/src/components/module-viewer/SideBarModuleOverview.tsx

```diff
@@ -104,8 +104,15 @@ const SideBarModuleOverview = ({
         <LessonItem
           editable={editable}
           text={lesson.title}
+          completed={
+            !!completedLessons.size &&
+            completedLessons.has(module.lessons[index])
+          }
+          isCurrent={
+            module.lessons[completedLessons.size] === module.lessons[index]
```

Comment:
Is there a particular reason to index into module.lessons as opposed to just checking if `index === completedLessons.size`?